[2017-09-26 04:55:44] lumen.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Call to a member function isValid() on null in /Users/user/selfie/app/Http/Controllers/ImagesController.php:43
Stack trace:
#0 [internal function]: App\Http\Controllers\ImagesController->createImage(Object(Illuminate\Http\Request), '1')
#1 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#2 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#3 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#4 /Users/user/selfie/vendor/illuminate/container/Container.php(549): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#5 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#6 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#7 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\ImagesController), 'createImage', Array)
#8 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#9 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#10 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#11 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#12 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#13 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#14 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#15 /Users/user/selfie/public/index.php(28): Laravel\Lumen\Application->run()
#16 {main}  
[2017-09-27 01:23:01] lumen.ERROR: PDOException: SQLSTATE[HY000] [2002] Connection refused in /Users/user/selfie/vendor/illuminate/database/Connectors/Connector.php:67
Stack trace:
#0 /Users/user/selfie/vendor/illuminate/database/Connectors/Connector.php(67): PDO->__construct('mysql:host=127....', 'root', '', Array)
#1 /Users/user/selfie/vendor/illuminate/database/Connectors/Connector.php(43): Illuminate\Database\Connectors\Connector->createPdoConnection('mysql:host=127....', 'root', '', Array)
#2 /Users/user/selfie/vendor/illuminate/database/Connectors/MySqlConnector.php(24): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=127....', Array, Array)
#3 /Users/user/selfie/vendor/illuminate/database/Connectors/ConnectionFactory.php(183): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#4 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#5 /Users/user/selfie/vendor/illuminate/database/Connection.php(913): call_user_func(Object(Closure))
#6 /Users/user/selfie/vendor/illuminate/database/Connection.php(938): Illuminate\Database\Connection->getPdo()
#7 /Users/user/selfie/vendor/illuminate/database/Connection.php(399): Illuminate\Database\Connection->getReadPdo()
#8 /Users/user/selfie/vendor/illuminate/database/Connection.php(325): Illuminate\Database\Connection->getPdoForSelect(true)
#9 /Users/user/selfie/vendor/illuminate/database/Connection.php(657): Illuminate\Database\Connection->Illuminate\Database\{closure}('select * from `...', Array)
#10 /Users/user/selfie/vendor/illuminate/database/Connection.php(624): Illuminate\Database\Connection->runQueryCallback('select * from `...', Array, Object(Closure))
#11 /Users/user/selfie/vendor/illuminate/database/Connection.php(333): Illuminate\Database\Connection->run('select * from `...', Array, Object(Closure))
#12 /Users/user/selfie/vendor/illuminate/database/Query/Builder.php(1705): Illuminate\Database\Connection->select('select * from `...', Array, true)
#13 /Users/user/selfie/vendor/illuminate/database/Query/Builder.php(1690): Illuminate\Database\Query\Builder->runSelect()
#14 /Users/user/selfie/vendor/illuminate/database/Eloquent/Builder.php(479): Illuminate\Database\Query\Builder->get(Array)
#15 /Users/user/selfie/vendor/illuminate/database/Eloquent/Builder.php(463): Illuminate\Database\Eloquent\Builder->getModels(Array)
#16 /Users/user/selfie/vendor/illuminate/database/Concerns/BuildsQueries.php(77): Illuminate\Database\Eloquent\Builder->get(Array)
#17 /Users/user/selfie/vendor/illuminate/database/Eloquent/Builder.php(289): Illuminate\Database\Eloquent\Builder->first(Array)
#18 /Users/user/selfie/vendor/illuminate/database/Eloquent/Model.php(1421): Illuminate\Database\Eloquent\Builder->find('2')
#19 /Users/user/selfie/vendor/illuminate/database/Eloquent/Model.php(1438): Illuminate\Database\Eloquent\Model->__call('find', Array)
#20 /Users/user/selfie/app/Http/Controllers/ProjectsController.php(20): Illuminate\Database\Eloquent\Model::__callStatic('find', Array)
#21 [internal function]: App\Http\Controllers\ProjectsController->getProject('2')
#22 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#23 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#24 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#25 /Users/user/selfie/vendor/illuminate/container/Container.php(549): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#26 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#27 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#28 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\ProjectsController), 'getProject', Array)
#29 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#30 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#31 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#32 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#33 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#34 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#35 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#36 /Users/user/selfie/public/index.php(28): Laravel\Lumen\Application->run()
#37 {main}  
[2017-09-27 01:23:01] lumen.ERROR: PDOException: SQLSTATE[HY000] [2002] Connection refused in /Users/user/selfie/vendor/illuminate/database/Connectors/Connector.php:67
Stack trace:
#0 /Users/user/selfie/vendor/illuminate/database/Connectors/Connector.php(67): PDO->__construct('mysql:host=127....', 'root', '', Array)
#1 /Users/user/selfie/vendor/illuminate/database/Connectors/Connector.php(43): Illuminate\Database\Connectors\Connector->createPdoConnection('mysql:host=127....', 'root', '', Array)
#2 /Users/user/selfie/vendor/illuminate/database/Connectors/MySqlConnector.php(24): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=127....', Array, Array)
#3 /Users/user/selfie/vendor/illuminate/database/Connectors/ConnectionFactory.php(183): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#4 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#5 /Users/user/selfie/vendor/illuminate/database/Connection.php(913): call_user_func(Object(Closure))
#6 /Users/user/selfie/vendor/illuminate/database/Connection.php(938): Illuminate\Database\Connection->getPdo()
#7 /Users/user/selfie/vendor/illuminate/database/Connection.php(399): Illuminate\Database\Connection->getReadPdo()
#8 /Users/user/selfie/vendor/illuminate/database/Connection.php(325): Illuminate\Database\Connection->getPdoForSelect(true)
#9 /Users/user/selfie/vendor/illuminate/database/Connection.php(657): Illuminate\Database\Connection->Illuminate\Database\{closure}('select * from `...', Array)
#10 /Users/user/selfie/vendor/illuminate/database/Connection.php(624): Illuminate\Database\Connection->runQueryCallback('select * from `...', Array, Object(Closure))
#11 /Users/user/selfie/vendor/illuminate/database/Connection.php(333): Illuminate\Database\Connection->run('select * from `...', Array, Object(Closure))
#12 /Users/user/selfie/vendor/illuminate/database/Query/Builder.php(1705): Illuminate\Database\Connection->select('select * from `...', Array, true)
#13 /Users/user/selfie/vendor/illuminate/database/Query/Builder.php(1690): Illuminate\Database\Query\Builder->runSelect()
#14 /Users/user/selfie/vendor/illuminate/database/Eloquent/Builder.php(479): Illuminate\Database\Query\Builder->get(Array)
#15 /Users/user/selfie/vendor/illuminate/database/Eloquent/Builder.php(463): Illuminate\Database\Eloquent\Builder->getModels(Array)
#16 /Users/user/selfie/vendor/illuminate/database/Concerns/BuildsQueries.php(77): Illuminate\Database\Eloquent\Builder->get(Array)
#17 /Users/user/selfie/vendor/illuminate/database/Eloquent/Builder.php(289): Illuminate\Database\Eloquent\Builder->first(Array)
#18 /Users/user/selfie/vendor/illuminate/database/Eloquent/Model.php(1421): Illuminate\Database\Eloquent\Builder->find('2')
#19 /Users/user/selfie/vendor/illuminate/database/Eloquent/Model.php(1438): Illuminate\Database\Eloquent\Model->__call('find', Array)
#20 /Users/user/selfie/app/Http/Controllers/ProjectsController.php(20): Illuminate\Database\Eloquent\Model::__callStatic('find', Array)
#21 [internal function]: App\Http\Controllers\ProjectsController->getProject('2')
#22 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#23 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#24 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#25 /Users/user/selfie/vendor/illuminate/container/Container.php(549): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#26 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#27 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#28 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\ProjectsController), 'getProject', Array)
#29 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#30 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#31 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#32 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#33 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#34 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#35 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#36 /Users/user/selfie/public/index.php(28): Laravel\Lumen\Application->run()
#37 {main}

Next Illuminate\Database\QueryException: SQLSTATE[HY000] [2002] Connection refused (SQL: select * from `projects` where `projects`.`id` = 2 limit 1) in /Users/user/selfie/vendor/illuminate/database/Connection.php:664
Stack trace:
#0 /Users/user/selfie/vendor/illuminate/database/Connection.php(624): Illuminate\Database\Connection->runQueryCallback('select * from `...', Array, Object(Closure))
#1 /Users/user/selfie/vendor/illuminate/database/Connection.php(333): Illuminate\Database\Connection->run('select * from `...', Array, Object(Closure))
#2 /Users/user/selfie/vendor/illuminate/database/Query/Builder.php(1705): Illuminate\Database\Connection->select('select * from `...', Array, true)
#3 /Users/user/selfie/vendor/illuminate/database/Query/Builder.php(1690): Illuminate\Database\Query\Builder->runSelect()
#4 /Users/user/selfie/vendor/illuminate/database/Eloquent/Builder.php(479): Illuminate\Database\Query\Builder->get(Array)
#5 /Users/user/selfie/vendor/illuminate/database/Eloquent/Builder.php(463): Illuminate\Database\Eloquent\Builder->getModels(Array)
#6 /Users/user/selfie/vendor/illuminate/database/Concerns/BuildsQueries.php(77): Illuminate\Database\Eloquent\Builder->get(Array)
#7 /Users/user/selfie/vendor/illuminate/database/Eloquent/Builder.php(289): Illuminate\Database\Eloquent\Builder->first(Array)
#8 /Users/user/selfie/vendor/illuminate/database/Eloquent/Model.php(1421): Illuminate\Database\Eloquent\Builder->find('2')
#9 /Users/user/selfie/vendor/illuminate/database/Eloquent/Model.php(1438): Illuminate\Database\Eloquent\Model->__call('find', Array)
#10 /Users/user/selfie/app/Http/Controllers/ProjectsController.php(20): Illuminate\Database\Eloquent\Model::__callStatic('find', Array)
#11 [internal function]: App\Http\Controllers\ProjectsController->getProject('2')
#12 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#13 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#14 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#15 /Users/user/selfie/vendor/illuminate/container/Container.php(549): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#16 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#17 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#18 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\ProjectsController), 'getProject', Array)
#19 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#20 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#21 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#22 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#23 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#24 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#25 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#26 /Users/user/selfie/public/index.php(28): Laravel\Lumen\Application->run()
#27 {main}  
[2017-09-27 01:23:33] lumen.ERROR: PDOException: SQLSTATE[HY000] [2002] Connection refused in /Users/user/selfie/vendor/illuminate/database/Connectors/Connector.php:67
Stack trace:
#0 /Users/user/selfie/vendor/illuminate/database/Connectors/Connector.php(67): PDO->__construct('mysql:host=127....', 'root', '', Array)
#1 /Users/user/selfie/vendor/illuminate/database/Connectors/Connector.php(43): Illuminate\Database\Connectors\Connector->createPdoConnection('mysql:host=127....', 'root', '', Array)
#2 /Users/user/selfie/vendor/illuminate/database/Connectors/MySqlConnector.php(24): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=127....', Array, Array)
#3 /Users/user/selfie/vendor/illuminate/database/Connectors/ConnectionFactory.php(183): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#4 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#5 /Users/user/selfie/vendor/illuminate/database/Connection.php(913): call_user_func(Object(Closure))
#6 /Users/user/selfie/vendor/illuminate/database/Connection.php(938): Illuminate\Database\Connection->getPdo()
#7 /Users/user/selfie/vendor/illuminate/database/Connection.php(399): Illuminate\Database\Connection->getReadPdo()
#8 /Users/user/selfie/vendor/illuminate/database/Connection.php(325): Illuminate\Database\Connection->getPdoForSelect(true)
#9 /Users/user/selfie/vendor/illuminate/database/Connection.php(657): Illuminate\Database\Connection->Illuminate\Database\{closure}('select * from `...', Array)
#10 /Users/user/selfie/vendor/illuminate/database/Connection.php(624): Illuminate\Database\Connection->runQueryCallback('select * from `...', Array, Object(Closure))
#11 /Users/user/selfie/vendor/illuminate/database/Connection.php(333): Illuminate\Database\Connection->run('select * from `...', Array, Object(Closure))
#12 /Users/user/selfie/vendor/illuminate/database/Query/Builder.php(1705): Illuminate\Database\Connection->select('select * from `...', Array, true)
#13 /Users/user/selfie/vendor/illuminate/database/Query/Builder.php(1690): Illuminate\Database\Query\Builder->runSelect()
#14 /Users/user/selfie/vendor/illuminate/database/Eloquent/Builder.php(479): Illuminate\Database\Query\Builder->get(Array)
#15 /Users/user/selfie/vendor/illuminate/database/Eloquent/Builder.php(463): Illuminate\Database\Eloquent\Builder->getModels(Array)
#16 /Users/user/selfie/vendor/illuminate/database/Concerns/BuildsQueries.php(77): Illuminate\Database\Eloquent\Builder->get(Array)
#17 /Users/user/selfie/vendor/illuminate/database/Eloquent/Builder.php(289): Illuminate\Database\Eloquent\Builder->first(Array)
#18 /Users/user/selfie/vendor/illuminate/database/Eloquent/Model.php(1421): Illuminate\Database\Eloquent\Builder->find('2')
#19 /Users/user/selfie/vendor/illuminate/database/Eloquent/Model.php(1438): Illuminate\Database\Eloquent\Model->__call('find', Array)
#20 /Users/user/selfie/app/Http/Controllers/ProjectsController.php(20): Illuminate\Database\Eloquent\Model::__callStatic('find', Array)
#21 [internal function]: App\Http\Controllers\ProjectsController->getProject('2')
#22 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#23 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#24 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#25 /Users/user/selfie/vendor/illuminate/container/Container.php(549): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#26 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#27 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#28 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\ProjectsController), 'getProject', Array)
#29 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#30 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#31 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#32 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#33 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#34 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#35 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#36 /Users/user/selfie/public/index.php(28): Laravel\Lumen\Application->run()
#37 {main}  
[2017-09-27 01:23:33] lumen.ERROR: PDOException: SQLSTATE[HY000] [2002] Connection refused in /Users/user/selfie/vendor/illuminate/database/Connectors/Connector.php:67
Stack trace:
#0 /Users/user/selfie/vendor/illuminate/database/Connectors/Connector.php(67): PDO->__construct('mysql:host=127....', 'root', '', Array)
#1 /Users/user/selfie/vendor/illuminate/database/Connectors/Connector.php(43): Illuminate\Database\Connectors\Connector->createPdoConnection('mysql:host=127....', 'root', '', Array)
#2 /Users/user/selfie/vendor/illuminate/database/Connectors/MySqlConnector.php(24): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=127....', Array, Array)
#3 /Users/user/selfie/vendor/illuminate/database/Connectors/ConnectionFactory.php(183): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#4 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#5 /Users/user/selfie/vendor/illuminate/database/Connection.php(913): call_user_func(Object(Closure))
#6 /Users/user/selfie/vendor/illuminate/database/Connection.php(938): Illuminate\Database\Connection->getPdo()
#7 /Users/user/selfie/vendor/illuminate/database/Connection.php(399): Illuminate\Database\Connection->getReadPdo()
#8 /Users/user/selfie/vendor/illuminate/database/Connection.php(325): Illuminate\Database\Connection->getPdoForSelect(true)
#9 /Users/user/selfie/vendor/illuminate/database/Connection.php(657): Illuminate\Database\Connection->Illuminate\Database\{closure}('select * from `...', Array)
#10 /Users/user/selfie/vendor/illuminate/database/Connection.php(624): Illuminate\Database\Connection->runQueryCallback('select * from `...', Array, Object(Closure))
#11 /Users/user/selfie/vendor/illuminate/database/Connection.php(333): Illuminate\Database\Connection->run('select * from `...', Array, Object(Closure))
#12 /Users/user/selfie/vendor/illuminate/database/Query/Builder.php(1705): Illuminate\Database\Connection->select('select * from `...', Array, true)
#13 /Users/user/selfie/vendor/illuminate/database/Query/Builder.php(1690): Illuminate\Database\Query\Builder->runSelect()
#14 /Users/user/selfie/vendor/illuminate/database/Eloquent/Builder.php(479): Illuminate\Database\Query\Builder->get(Array)
#15 /Users/user/selfie/vendor/illuminate/database/Eloquent/Builder.php(463): Illuminate\Database\Eloquent\Builder->getModels(Array)
#16 /Users/user/selfie/vendor/illuminate/database/Concerns/BuildsQueries.php(77): Illuminate\Database\Eloquent\Builder->get(Array)
#17 /Users/user/selfie/vendor/illuminate/database/Eloquent/Builder.php(289): Illuminate\Database\Eloquent\Builder->first(Array)
#18 /Users/user/selfie/vendor/illuminate/database/Eloquent/Model.php(1421): Illuminate\Database\Eloquent\Builder->find('2')
#19 /Users/user/selfie/vendor/illuminate/database/Eloquent/Model.php(1438): Illuminate\Database\Eloquent\Model->__call('find', Array)
#20 /Users/user/selfie/app/Http/Controllers/ProjectsController.php(20): Illuminate\Database\Eloquent\Model::__callStatic('find', Array)
#21 [internal function]: App\Http\Controllers\ProjectsController->getProject('2')
#22 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#23 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#24 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#25 /Users/user/selfie/vendor/illuminate/container/Container.php(549): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#26 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#27 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#28 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\ProjectsController), 'getProject', Array)
#29 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#30 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#31 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#32 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#33 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#34 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#35 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#36 /Users/user/selfie/public/index.php(28): Laravel\Lumen\Application->run()
#37 {main}

Next Illuminate\Database\QueryException: SQLSTATE[HY000] [2002] Connection refused (SQL: select * from `projects` where `projects`.`id` = 2 limit 1) in /Users/user/selfie/vendor/illuminate/database/Connection.php:664
Stack trace:
#0 /Users/user/selfie/vendor/illuminate/database/Connection.php(624): Illuminate\Database\Connection->runQueryCallback('select * from `...', Array, Object(Closure))
#1 /Users/user/selfie/vendor/illuminate/database/Connection.php(333): Illuminate\Database\Connection->run('select * from `...', Array, Object(Closure))
#2 /Users/user/selfie/vendor/illuminate/database/Query/Builder.php(1705): Illuminate\Database\Connection->select('select * from `...', Array, true)
#3 /Users/user/selfie/vendor/illuminate/database/Query/Builder.php(1690): Illuminate\Database\Query\Builder->runSelect()
#4 /Users/user/selfie/vendor/illuminate/database/Eloquent/Builder.php(479): Illuminate\Database\Query\Builder->get(Array)
#5 /Users/user/selfie/vendor/illuminate/database/Eloquent/Builder.php(463): Illuminate\Database\Eloquent\Builder->getModels(Array)
#6 /Users/user/selfie/vendor/illuminate/database/Concerns/BuildsQueries.php(77): Illuminate\Database\Eloquent\Builder->get(Array)
#7 /Users/user/selfie/vendor/illuminate/database/Eloquent/Builder.php(289): Illuminate\Database\Eloquent\Builder->first(Array)
#8 /Users/user/selfie/vendor/illuminate/database/Eloquent/Model.php(1421): Illuminate\Database\Eloquent\Builder->find('2')
#9 /Users/user/selfie/vendor/illuminate/database/Eloquent/Model.php(1438): Illuminate\Database\Eloquent\Model->__call('find', Array)
#10 /Users/user/selfie/app/Http/Controllers/ProjectsController.php(20): Illuminate\Database\Eloquent\Model::__callStatic('find', Array)
#11 [internal function]: App\Http\Controllers\ProjectsController->getProject('2')
#12 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#13 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#14 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#15 /Users/user/selfie/vendor/illuminate/container/Container.php(549): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#16 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#17 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#18 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\ProjectsController), 'getProject', Array)
#19 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#20 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#21 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#22 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#23 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#24 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#25 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#26 /Users/user/selfie/public/index.php(28): Laravel\Lumen\Application->run()
#27 {main}  
[2017-09-27 01:23:35] lumen.ERROR: PDOException: SQLSTATE[HY000] [2002] Connection refused in /Users/user/selfie/vendor/illuminate/database/Connectors/Connector.php:67
Stack trace:
#0 /Users/user/selfie/vendor/illuminate/database/Connectors/Connector.php(67): PDO->__construct('mysql:host=127....', 'root', '', Array)
#1 /Users/user/selfie/vendor/illuminate/database/Connectors/Connector.php(43): Illuminate\Database\Connectors\Connector->createPdoConnection('mysql:host=127....', 'root', '', Array)
#2 /Users/user/selfie/vendor/illuminate/database/Connectors/MySqlConnector.php(24): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=127....', Array, Array)
#3 /Users/user/selfie/vendor/illuminate/database/Connectors/ConnectionFactory.php(183): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#4 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#5 /Users/user/selfie/vendor/illuminate/database/Connection.php(913): call_user_func(Object(Closure))
#6 /Users/user/selfie/vendor/illuminate/database/Connection.php(938): Illuminate\Database\Connection->getPdo()
#7 /Users/user/selfie/vendor/illuminate/database/Connection.php(399): Illuminate\Database\Connection->getReadPdo()
#8 /Users/user/selfie/vendor/illuminate/database/Connection.php(325): Illuminate\Database\Connection->getPdoForSelect(true)
#9 /Users/user/selfie/vendor/illuminate/database/Connection.php(657): Illuminate\Database\Connection->Illuminate\Database\{closure}('select * from `...', Array)
#10 /Users/user/selfie/vendor/illuminate/database/Connection.php(624): Illuminate\Database\Connection->runQueryCallback('select * from `...', Array, Object(Closure))
#11 /Users/user/selfie/vendor/illuminate/database/Connection.php(333): Illuminate\Database\Connection->run('select * from `...', Array, Object(Closure))
#12 /Users/user/selfie/vendor/illuminate/database/Query/Builder.php(1705): Illuminate\Database\Connection->select('select * from `...', Array, true)
#13 /Users/user/selfie/vendor/illuminate/database/Query/Builder.php(1690): Illuminate\Database\Query\Builder->runSelect()
#14 /Users/user/selfie/vendor/illuminate/database/Eloquent/Builder.php(479): Illuminate\Database\Query\Builder->get(Array)
#15 /Users/user/selfie/vendor/illuminate/database/Eloquent/Builder.php(463): Illuminate\Database\Eloquent\Builder->getModels(Array)
#16 /Users/user/selfie/vendor/illuminate/database/Concerns/BuildsQueries.php(77): Illuminate\Database\Eloquent\Builder->get(Array)
#17 /Users/user/selfie/vendor/illuminate/database/Eloquent/Builder.php(289): Illuminate\Database\Eloquent\Builder->first(Array)
#18 /Users/user/selfie/vendor/illuminate/database/Eloquent/Model.php(1421): Illuminate\Database\Eloquent\Builder->find('2')
#19 /Users/user/selfie/vendor/illuminate/database/Eloquent/Model.php(1438): Illuminate\Database\Eloquent\Model->__call('find', Array)
#20 /Users/user/selfie/app/Http/Controllers/ProjectsController.php(20): Illuminate\Database\Eloquent\Model::__callStatic('find', Array)
#21 [internal function]: App\Http\Controllers\ProjectsController->getProject('2')
#22 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#23 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#24 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#25 /Users/user/selfie/vendor/illuminate/container/Container.php(549): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#26 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#27 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#28 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\ProjectsController), 'getProject', Array)
#29 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#30 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#31 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#32 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#33 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#34 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#35 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#36 /Users/user/selfie/public/index.php(28): Laravel\Lumen\Application->run()
#37 {main}  
[2017-09-27 01:23:35] lumen.ERROR: PDOException: SQLSTATE[HY000] [2002] Connection refused in /Users/user/selfie/vendor/illuminate/database/Connectors/Connector.php:67
Stack trace:
#0 /Users/user/selfie/vendor/illuminate/database/Connectors/Connector.php(67): PDO->__construct('mysql:host=127....', 'root', '', Array)
#1 /Users/user/selfie/vendor/illuminate/database/Connectors/Connector.php(43): Illuminate\Database\Connectors\Connector->createPdoConnection('mysql:host=127....', 'root', '', Array)
#2 /Users/user/selfie/vendor/illuminate/database/Connectors/MySqlConnector.php(24): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=127....', Array, Array)
#3 /Users/user/selfie/vendor/illuminate/database/Connectors/ConnectionFactory.php(183): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#4 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#5 /Users/user/selfie/vendor/illuminate/database/Connection.php(913): call_user_func(Object(Closure))
#6 /Users/user/selfie/vendor/illuminate/database/Connection.php(938): Illuminate\Database\Connection->getPdo()
#7 /Users/user/selfie/vendor/illuminate/database/Connection.php(399): Illuminate\Database\Connection->getReadPdo()
#8 /Users/user/selfie/vendor/illuminate/database/Connection.php(325): Illuminate\Database\Connection->getPdoForSelect(true)
#9 /Users/user/selfie/vendor/illuminate/database/Connection.php(657): Illuminate\Database\Connection->Illuminate\Database\{closure}('select * from `...', Array)
#10 /Users/user/selfie/vendor/illuminate/database/Connection.php(624): Illuminate\Database\Connection->runQueryCallback('select * from `...', Array, Object(Closure))
#11 /Users/user/selfie/vendor/illuminate/database/Connection.php(333): Illuminate\Database\Connection->run('select * from `...', Array, Object(Closure))
#12 /Users/user/selfie/vendor/illuminate/database/Query/Builder.php(1705): Illuminate\Database\Connection->select('select * from `...', Array, true)
#13 /Users/user/selfie/vendor/illuminate/database/Query/Builder.php(1690): Illuminate\Database\Query\Builder->runSelect()
#14 /Users/user/selfie/vendor/illuminate/database/Eloquent/Builder.php(479): Illuminate\Database\Query\Builder->get(Array)
#15 /Users/user/selfie/vendor/illuminate/database/Eloquent/Builder.php(463): Illuminate\Database\Eloquent\Builder->getModels(Array)
#16 /Users/user/selfie/vendor/illuminate/database/Concerns/BuildsQueries.php(77): Illuminate\Database\Eloquent\Builder->get(Array)
#17 /Users/user/selfie/vendor/illuminate/database/Eloquent/Builder.php(289): Illuminate\Database\Eloquent\Builder->first(Array)
#18 /Users/user/selfie/vendor/illuminate/database/Eloquent/Model.php(1421): Illuminate\Database\Eloquent\Builder->find('2')
#19 /Users/user/selfie/vendor/illuminate/database/Eloquent/Model.php(1438): Illuminate\Database\Eloquent\Model->__call('find', Array)
#20 /Users/user/selfie/app/Http/Controllers/ProjectsController.php(20): Illuminate\Database\Eloquent\Model::__callStatic('find', Array)
#21 [internal function]: App\Http\Controllers\ProjectsController->getProject('2')
#22 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#23 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#24 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#25 /Users/user/selfie/vendor/illuminate/container/Container.php(549): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#26 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#27 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#28 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\ProjectsController), 'getProject', Array)
#29 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#30 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#31 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#32 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#33 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#34 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#35 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#36 /Users/user/selfie/public/index.php(28): Laravel\Lumen\Application->run()
#37 {main}

Next Illuminate\Database\QueryException: SQLSTATE[HY000] [2002] Connection refused (SQL: select * from `projects` where `projects`.`id` = 2 limit 1) in /Users/user/selfie/vendor/illuminate/database/Connection.php:664
Stack trace:
#0 /Users/user/selfie/vendor/illuminate/database/Connection.php(624): Illuminate\Database\Connection->runQueryCallback('select * from `...', Array, Object(Closure))
#1 /Users/user/selfie/vendor/illuminate/database/Connection.php(333): Illuminate\Database\Connection->run('select * from `...', Array, Object(Closure))
#2 /Users/user/selfie/vendor/illuminate/database/Query/Builder.php(1705): Illuminate\Database\Connection->select('select * from `...', Array, true)
#3 /Users/user/selfie/vendor/illuminate/database/Query/Builder.php(1690): Illuminate\Database\Query\Builder->runSelect()
#4 /Users/user/selfie/vendor/illuminate/database/Eloquent/Builder.php(479): Illuminate\Database\Query\Builder->get(Array)
#5 /Users/user/selfie/vendor/illuminate/database/Eloquent/Builder.php(463): Illuminate\Database\Eloquent\Builder->getModels(Array)
#6 /Users/user/selfie/vendor/illuminate/database/Concerns/BuildsQueries.php(77): Illuminate\Database\Eloquent\Builder->get(Array)
#7 /Users/user/selfie/vendor/illuminate/database/Eloquent/Builder.php(289): Illuminate\Database\Eloquent\Builder->first(Array)
#8 /Users/user/selfie/vendor/illuminate/database/Eloquent/Model.php(1421): Illuminate\Database\Eloquent\Builder->find('2')
#9 /Users/user/selfie/vendor/illuminate/database/Eloquent/Model.php(1438): Illuminate\Database\Eloquent\Model->__call('find', Array)
#10 /Users/user/selfie/app/Http/Controllers/ProjectsController.php(20): Illuminate\Database\Eloquent\Model::__callStatic('find', Array)
#11 [internal function]: App\Http\Controllers\ProjectsController->getProject('2')
#12 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#13 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#14 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#15 /Users/user/selfie/vendor/illuminate/container/Container.php(549): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#16 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#17 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#18 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\ProjectsController), 'getProject', Array)
#19 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#20 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#21 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#22 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#23 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#24 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#25 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#26 /Users/user/selfie/public/index.php(28): Laravel\Lumen\Application->run()
#27 {main}  
[2017-09-27 03:10:26] lumen.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Call to a member function isValid() on null in /Users/user/selfie/app/Http/Controllers/ImagesController.php:43
Stack trace:
#0 [internal function]: App\Http\Controllers\ImagesController->createImage(Object(Illuminate\Http\Request), '2')
#1 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#2 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#3 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#4 /Users/user/selfie/vendor/illuminate/container/Container.php(549): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#5 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#6 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#7 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\ImagesController), 'createImage', Array)
#8 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#9 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#10 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#11 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#12 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#13 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#14 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#15 /Users/user/selfie/public/index.php(28): Laravel\Lumen\Application->run()
#16 {main}  
[2017-09-27 03:10:58] lumen.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Call to a member function isValid() on null in /Users/user/selfie/app/Http/Controllers/ImagesController.php:43
Stack trace:
#0 [internal function]: App\Http\Controllers\ImagesController->createImage(Object(Illuminate\Http\Request), '2')
#1 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#2 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#3 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#4 /Users/user/selfie/vendor/illuminate/container/Container.php(549): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#5 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#6 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#7 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\ImagesController), 'createImage', Array)
#8 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#9 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#10 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#11 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#12 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#13 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#14 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#15 /Users/user/selfie/public/index.php(28): Laravel\Lumen\Application->run()
#16 {main}  
[2017-09-27 03:16:42] lumen.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Call to a member function isValid() on null in /Users/user/selfie/app/Http/Controllers/ImagesController.php:43
Stack trace:
#0 [internal function]: App\Http\Controllers\ImagesController->createImage(Object(Illuminate\Http\Request), '2')
#1 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#2 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#3 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#4 /Users/user/selfie/vendor/illuminate/container/Container.php(549): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#5 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#6 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#7 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\ImagesController), 'createImage', Array)
#8 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#9 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#10 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#11 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#12 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#13 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#14 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#15 /Users/user/selfie/public/index.php(28): Laravel\Lumen\Application->run()
#16 {main}  
[2017-09-27 04:44:13] lumen.ERROR: Symfony\Component\HttpFoundation\File\Exception\FileException: The file "url" exceeds your upload_max_filesize ini directive (limit is 2048 KiB). in /Users/user/selfie/vendor/symfony/http-foundation/File/UploadedFile.php:235
Stack trace:
#0 /Users/user/selfie/app/Http/Controllers/ImagesController.php(51): Symfony\Component\HttpFoundation\File\UploadedFile->move('/Users/user/sel...', '27_09_17_04_44u...')
#1 [internal function]: App\Http\Controllers\ImagesController->createImage(Object(Illuminate\Http\Request), '2')
#2 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#3 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#4 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#5 /Users/user/selfie/vendor/illuminate/container/Container.php(549): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#6 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#7 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#8 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\ImagesController), 'createImage', Array)
#9 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#10 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#11 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#12 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#13 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#14 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#15 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#16 /Users/user/selfie/public/index.php(28): Laravel\Lumen\Application->run()
#17 {main}  
[2017-09-27 04:49:22] lumen.ERROR: Symfony\Component\HttpFoundation\File\Exception\FileException: The file "110-1.jpg" exceeds your upload_max_filesize ini directive (limit is 2048 KiB). in /Users/user/selfie/vendor/symfony/http-foundation/File/UploadedFile.php:235
Stack trace:
#0 /Users/user/selfie/app/Http/Controllers/ImagesController.php(51): Symfony\Component\HttpFoundation\File\UploadedFile->move('/Users/user/sel...', '27_09_17_04_491...')
#1 [internal function]: App\Http\Controllers\ImagesController->createImage(Object(Illuminate\Http\Request), '2')
#2 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#3 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#4 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#5 /Users/user/selfie/vendor/illuminate/container/Container.php(549): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#6 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#7 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#8 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\ImagesController), 'createImage', Array)
#9 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#10 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#11 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#12 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#13 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#14 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#15 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#16 /Users/user/selfie/public/index.php(28): Laravel\Lumen\Application->run()
#17 {main}  
[2017-09-27 04:50:21] lumen.ERROR: Symfony\Component\HttpFoundation\File\Exception\FileException: The file "110-1.jpg" exceeds your upload_max_filesize ini directive (limit is 2048 KiB). in /Users/user/selfie/vendor/symfony/http-foundation/File/UploadedFile.php:235
Stack trace:
#0 /Users/user/selfie/app/Http/Controllers/ImagesController.php(51): Symfony\Component\HttpFoundation\File\UploadedFile->move('/Users/user/sel...', '27_09_17_04_501...')
#1 [internal function]: App\Http\Controllers\ImagesController->createImage(Object(Illuminate\Http\Request), '2')
#2 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#3 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#4 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#5 /Users/user/selfie/vendor/illuminate/container/Container.php(549): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#6 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#7 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#8 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\ImagesController), 'createImage', Array)
#9 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#10 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#11 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#12 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#13 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#14 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#15 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#16 /Users/user/selfie/public/index.php(28): Laravel\Lumen\Application->run()
#17 {main}  
[2017-09-27 04:57:43] lumen.ERROR: Symfony\Component\HttpFoundation\File\Exception\FileException: The file "110-1.jpg" exceeds your upload_max_filesize ini directive (limit is 2048 KiB). in /Users/user/selfie/vendor/symfony/http-foundation/File/UploadedFile.php:235
Stack trace:
#0 /Users/user/selfie/app/Http/Controllers/ImagesController.php(51): Symfony\Component\HttpFoundation\File\UploadedFile->move('/Users/user/sel...', '27_09_17_04_571...')
#1 [internal function]: App\Http\Controllers\ImagesController->createImage(Object(Illuminate\Http\Request), '2')
#2 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#3 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#4 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#5 /Users/user/selfie/vendor/illuminate/container/Container.php(549): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#6 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#7 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#8 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\ImagesController), 'createImage', Array)
#9 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#10 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#11 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#12 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#13 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#14 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#15 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#16 /Users/user/selfie/public/index.php(28): Laravel\Lumen\Application->run()
#17 {main}  
[2017-09-27 04:58:55] lumen.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Parse error: syntax error, unexpected 'else' (T_ELSE), expecting function (T_FUNCTION) or const (T_CONST) in /Users/user/selfie/app/Http/Controllers/ImagesController.php:62
Stack trace:
#0 /Users/user/selfie/vendor/composer/ClassLoader.php(322): Composer\Autoload\includeFile('/Users/user/sel...')
#1 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Http\\Contro...')
#2 [internal function]: spl_autoload_call('App\\Http\\Contro...')
#3 /Users/user/selfie/vendor/illuminate/container/Container.php(752): ReflectionClass->__construct('App\\Http\\Contro...')
#4 /Users/user/selfie/vendor/illuminate/container/Container.php(631): Illuminate\Container\Container->build('App\\Http\\Contro...')
#5 /Users/user/selfie/vendor/illuminate/container/Container.php(586): Illuminate\Container\Container->resolve('App\\Http\\Contro...', Array)
#6 /Users/user/selfie/vendor/laravel/lumen-framework/src/Application.php(230): Illuminate\Container\Container->make('App\\Http\\Contro...', Array)
#7 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(308): Laravel\Lumen\Application->make('App\\Http\\Contro...')
#8 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#9 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#10 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#11 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#12 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#13 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#14 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#15 /Users/user/selfie/public/index.php(28): Laravel\Lumen\Application->run()
#16 {main}  
[2017-09-27 04:59:03] lumen.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Parse error: syntax error, unexpected 'else' (T_ELSE), expecting function (T_FUNCTION) or const (T_CONST) in /Users/user/selfie/app/Http/Controllers/ImagesController.php:62
Stack trace:
#0 /Users/user/selfie/vendor/composer/ClassLoader.php(322): Composer\Autoload\includeFile('/Users/user/sel...')
#1 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Http\\Contro...')
#2 [internal function]: spl_autoload_call('App\\Http\\Contro...')
#3 /Users/user/selfie/vendor/illuminate/container/Container.php(752): ReflectionClass->__construct('App\\Http\\Contro...')
#4 /Users/user/selfie/vendor/illuminate/container/Container.php(631): Illuminate\Container\Container->build('App\\Http\\Contro...')
#5 /Users/user/selfie/vendor/illuminate/container/Container.php(586): Illuminate\Container\Container->resolve('App\\Http\\Contro...', Array)
#6 /Users/user/selfie/vendor/laravel/lumen-framework/src/Application.php(230): Illuminate\Container\Container->make('App\\Http\\Contro...', Array)
#7 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(308): Laravel\Lumen\Application->make('App\\Http\\Contro...')
#8 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#9 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#10 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#11 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#12 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#13 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#14 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#15 /Users/user/selfie/public/index.php(28): Laravel\Lumen\Application->run()
#16 {main}  
[2017-09-27 05:22:18] lumen.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Call to a member function isValid() on null in /Users/user/selfie/app/Http/Controllers/ImagesController.php:43
Stack trace:
#0 [internal function]: App\Http\Controllers\ImagesController->createImage(Object(Illuminate\Http\Request), '2')
#1 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#2 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#3 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#4 /Users/user/selfie/vendor/illuminate/container/Container.php(549): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#5 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#6 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#7 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\ImagesController), 'createImage', Array)
#8 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#9 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#10 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#11 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#12 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#13 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#14 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#15 /Users/user/selfie/public/index.php(28): Laravel\Lumen\Application->run()
#16 {main}  
[2017-09-27 05:23:29] lumen.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Call to a member function isValid() on null in /Users/user/selfie/app/Http/Controllers/ImagesController.php:43
Stack trace:
#0 [internal function]: App\Http\Controllers\ImagesController->createImage(Object(Illuminate\Http\Request), '2')
#1 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#2 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#3 /Users/user/selfie/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#4 /Users/user/selfie/vendor/illuminate/container/Container.php(549): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#5 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#6 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#7 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\ImagesController), 'createImage', Array)
#8 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#9 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#10 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#11 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#12 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#13 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#14 /Users/user/selfie/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#15 /Users/user/selfie/public/index.php(28): Laravel\Lumen\Application->run()
#16 {main}  
